### ISSUE TITLE
Support IE8 with attachEvent

### ISSUE THREAD

**Issue by cyounkins** [gsklee/ngStorage#35]
I haven't tested but the event listener isn't likely to work because window.addEventListener() is not available on IE8.

IE8 is the lowest version of IE supported by Angular, so I think ngStorage should support it as well.

Fix is to check if window.addEventListener exists, and if not, fall back to window.attachEvent.

Will try to send PR later today.

**Comment by RopoMen** [comment]
In my opinion is that ngStorage should update bower.json and set required angular version to 1.3 and up. I know that this hurts some people, but we cannot support IE8 forever. I know the pain also, because I'm currently working on a project where IE8 is required, but it doesn't mean that these kind of libs should always be made IE8 compatible.

I usually tell the customer that IE8 is not supported anymore and custom work costs this much of $$$, do you still want this? :) (because I do not like either on these "if(IE)" hax)

You can always fork this lib and make those IE8 fixes for your project.

**Comment by simison** [comment]
Angular supports only IE9+

https://docs.angularjs.org/guide/ie

**Comment by RopoMen** [comment]
Yes and no, because if you need to support IE8 then you use AngularJS 1.2.x series. That is why my suggestion was that ngStorage module should update bower.json to support minimum angular version 1.3, which would state that ngStorage is not even intending to support IE8. 
Now that is not clear, because bower.json is requiring minimum angular version ">=1.0.8" which covers also IE8 support.
